common/src/common.py

```diff
@@ -73,4 +73,12 @@ def diff_and_save(data, fname, verbose=True):
   with open(fname, 'w') as f:
     json.dump(data, f)
 
-  return diff
\ No newline at end of file
+  return diff
+
+def normalize_state(data):
+  ''' Sort data in-place for consistent ordering '''
+  for locale in data:
```

Comment:
Let's be consistent.  `locale` is a bad name: it's the field name.  Let's call this `datum` or `x`.

Reply:
updated variable name to datum